### PR TITLE
Added coverage for freeIPA LDAP auth

### PR DESCRIPTION
**Pull request by ntkathole**
Test results

```
 pytest tests/foreman/ui/test_ldapauthsource.py::LDAPAuthSourceTestCase::test_positive_create_withidm_org_and_loc
============================================================= test session starts =================================================================

collected 5 items


tests/foreman/ui/test_ldapauthsource.py .

================================================================= 1 passed =================================================================
```

### CODE REVIEW

**Comment by ntkathole** [comment]
Do not merge until gitlab PR 82

**Comment by ntkathole** [comment]
gitlab PR 82 merged

File: tests/foreman/ui/test_ldapauthsource.py [hunk 1]
```diff
@@ -153,7 +158,6 @@ def test_positive_create_withad_org_and_loc(self):
                         self.ldapauthsource.search(server_name)
                     )
 
-    @stubbed()
     @tier2
     def test_positive_create_withidm_org_and_loc(self):
```
Comment (abalakh) [inline tests/foreman/ui/test_ldapauthsource.py]: nitpick: /s/withidm/with_idm/ . Would be also nice to fix all 'withad' titles as well.

File: tests/foreman/ui/test_ldapauthsource.py [hunk 2]
```diff
@@ -19,7 +19,7 @@
 from robottelo.constants import LDAP_ATTR, LDAP_SERVER_TYPE
 from robottelo.datafactory import generate_strings_list
 from robottelo.decorators import (
-    skip_if_not_set, stubbed, tier1, tier2, upgrade)
+    skip_if_not_set, tier1, tier2, upgrade)
```
Comment (ldjebran) [inline tests/foreman/ui/test_ldapauthsource.py]: rewrite like like
```python
from robottelo.decorators import (
    skip_if_not_set,
    tier1,
    tier2,
    upgrade,
)
```
or like 
```python
from robottelo.decorators import skip_if_not_set, tier1, tier2, upgrade
```

File: tests/foreman/ui/test_ldapauthsource.py [hunk 3]
```diff
@@ -170,3 +174,27 @@ def test_positive_create_withidm_org_and_loc(self):
...
+                        server_type=LDAP_SERVER_TYPE['UI']['ipa'],
+                        login_name=LDAP_ATTR['login'],
+                        first_name=LDAP_ATTR['firstname'],
+                        surname=LDAP_ATTR['surname'],
+                        mail=LDAP_ATTR['mail'],
+                        account_user=self.ldap_ipa_user_name,
+                        account_passwd=self.ldap_ipa_user_passwd,
+                        account_basedn=self.ipa_base_dn,
+                        account_grpbasedn=self.ipa_group_base_dn,
+                        orgs=[entities.Organization().create().name],
```
Comment (ldjebran) [inline tests/foreman/ui/test_ldapauthsource.py]: why we have to create a new organization and location for each subtest?

**Comment by ntkathole** [comment]
@abalakh and @ldjebran please revisit...updated test results:


```
pytest tests/foreman/ui/test_ldapauthsource.py::LDAPAuthSourceTestCase::test_positive_create_with_ad_org_and_loc
========================================================== test session starts ==========================================================

collected 5 items


tests/foreman/ui/test_ldapauthsource.py .

======================================================= 1 passed =====================================================
```

```
 pytest tests/foreman/ui/test_ldapauthsource.py::LDAPAuthSourceTestCase::test_positive_create_with_idm_org_and_loc
========================================================== test session starts ==========================================================

collected 5 items

tests/foreman/ui/test_ldapauthsource.py .

======================================================= 1 passed =======================================================
```

**Pull request merged**